etc/freeoffice-textmaker

```diff
@@ -0,0 +1,48 @@
+# Firejail profile for freeoffice-textmaker
+# This file is overwritten after every install/update
+# Persistent local customizations
+include freeoffice-textmaker.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/SoftMaker
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+# include disable-xdg.inc
+
+# apparmor
+caps.drop all
+ipc-namespace
+machine-id
```

Comment:
breaks sound, because of `# nosound` it should be removed

Reply:
Sorry, I don't understand that remark. Why does it break sound if `nosound` is commented?

All other proposed changes added to the profiles.

Reply:
The why is a little difficult, but it is a fact that machine-id breaks the sound (If pulse is used).

Reply:
Okay, removed.